plugins/LilyPad/DualShock3.cpp

```diff
@@ -208,13 +208,13 @@ class DualShock3Device : public Device
     int index;
     HANDLE hFile;
     DS3Command sendState;
-    unsigned char getState[49];
+    unsigned char getState[49] = {};
     OVERLAPPED readop;
     OVERLAPPED writeop;
     int writeCount;
-    int lastWrite;
+    int lastWrite = NULL;
```

Comment:
IMO it's more readable if you use 0 instead of NULL when the type is not a pointer.

At the very least, I associate `NULL` with pointers.